ui/static/index.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
```

Comment:
I believe you need copyright & license header for every file.

Also, can you add a comment saying this file is used only for development? May be even clearer if it's named e.g., dev_index.html.

Reply:
Renamed to dev_index.html.
See PR #466 about licenses.

Reply:
Hm, actually webpack seems to expect the file to be called index.html (they have an "index" setting that sounded promising, but it doesn't do what we need). Added a comment at the top instead.